README.md

```diff
@@ -86,6 +86,16 @@ If you don't want to use multi-room or you only have one device, you can disable
 
 **Note:** Multi-room requires a network router that supports IP multicast/broadcast (most modern routers do).
 
+### Client Only Multi-room
+
+By default, balenaSound let's you connect to every device available. If you prefer to have one or multiple devices "hidden" in the network but still capable to play the multi-room audio you can enable the Client Only mode.
+
+Client Only mode only runs the services needed to synchronize and play the audio from the multi-room setup, and disables all other services. This makes for less clutter in the devices overview when connecting from Spotify, Airplay or Bluetooth and lowers the load on the devices running Client Only mode.
```

Comment:
`Client Only mode only runs the services needed to synchronize and play the audio that is being streamed from the multi-room master server; all other services will be disabled. This makes for less clutter in the devices overview when connecting from Spotify, Airplay or Bluetooth and lowers the load on the devices running Client Only mode.`

Reply:
Done!